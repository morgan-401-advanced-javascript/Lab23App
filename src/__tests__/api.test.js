import React from 'react';
import Api from '../components/api';
import {configure, mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

configure({adapter:new Adapter()});

describe('', () => {
    it('', () => {
        expect(true).toBeTruthy();
    });
});

describe('Api component', () => {
    it('renders correctly', () => {
        const page = renderer.create(<Api />).toJSON();
        expect(page).toMatchSnapshot();
    });
});

describe('Api Enzyme Testing', () => {
    it('has a Heading', () => {
        let component = mount(<Api />);
        expect(component.find('div h3').text()).toBe(
            'Words of Wisdom',
        );
    })});