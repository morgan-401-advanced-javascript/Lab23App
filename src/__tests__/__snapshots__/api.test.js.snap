// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Api component renders correctly 1`] = `
<div>
  <h3>
    Words of Wisdom
  </h3>
  <div>
    <p />
    <button
      className="btn btn-outline-primary"
      onClick={[Function]}
    >
      New Advice
    </button>
  </div>
</div>
`;
